sdk/include/psdk/dwmapi.h

```diff
@@ -211,6 +270,9 @@ DWMAPI DwmSetIconicLivePreviewBitmap(HWND, HBITMAP, POINT*, DWORD);
 DWMAPI DwmSetIconicThumbnail(HWND, HBITMAP, DWORD);
 DWMAPI DwmSetPresentParameters(HWND, DWM_PRESENT_PARAMETERS *);
 DWMAPI DwmSetWindowAttribute(HWND, DWORD, LPCVOID, DWORD);
+DWMAPI DwmShowContact(_In_ DWORD, _In_ DWM_SHOWCONTACT);
+DWMAPI DwmTetherContact(DWORD, BOOL, POINT);
+DWMPAI DwmTransitionOwnedWindow(HWND, DWMTRANSITION_OWNEDWINDOW_TARGET *);
```

Comment:
```suggestion
DWMAPI DwmTransitionOwnedWindow(HWND, DWMTRANSITION_OWNEDWINDOW_TARGET);
```
Typo fixes; see also https://learn.microsoft.com/en-us/windows/win32/api/dwmapi/nf-dwmapi-dwmtransitionownedwindow  for the second parameter type.